CNV/deploy-cnv.sh

```diff
@@ -95,5 +95,48 @@ done
 
 oc wait --for condition=ready pod -l app=kubevirt-node-labeller -n ${OPERATORS_NAMESPACE} --timeout=2400s
 
+echo "Setting up a bridge as the default interface"
+
+while [ "x" == "x$(oc get nodenetworkstate -o jsonpath='{.items[0].status.currentState.routes.running[?(@.destination=="0.0.0.0/0")].next-hop-interface}')" ]; do
+    sleep 10
+done
+
+default_iface=$(oc get nodenetworkstate -o jsonpath='{.items[0].status.currentState.routes.running[?(@.destination=="0.0.0.0/0")].next-hop-interface}')
+
+cat <<EOF | oc apply -f -
+apiVersion: nmstate.io/v1alpha1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: brext-policy
+spec:
+  desiredState:
+    interfaces:
+    - name: brext
+      type: linux-bridge
+      state: up
+      ipv4:
+        dhcp: true
+        enabled: true
+      ipv6:
+        dhcp: true
+        enabled: true
+      bridge:
+        options:
+          vlan-filtering: true
+          vlans:
+          - vlan-range-min: 1
+            vlan-range-max: 4094
+        port:
+        - name: ${default_interface}
+          vlans:
+          - vlan-range-min: 1
+            vlan-range-max: 4094
+EOF
+
+# Wait until the connectivity comes back
+sleep 10
+until oc get nodes 2> /dev/null; do
```

Comment:
Do you want to check if the nodes went back to ready as well?

Reply:
Done